detection-rules/attachment_pdf_file_fraudulent_cryptocurrency_exchange.yml

```
@@ -0,0 +1,22 @@
+name: "Attachment: PDF file with Link to Fake Bitcoin Exchange"
+description: |
+  Fraudulent message containing a PDF notification of unclaimed Bitcoin assets. 
+  The PDF file contains a link to a fake CryptoCurrency portal. Attempting to withdrawl
+  funds prompts the user to enter payment information
+references:
+  - "https://app.any.run/tasks/93d72f24-c021-48ad-a8eb-9b30c8660938"
+type: "rule"
+severity: "low"
+source: |
+   type.inbound 
+   and sender.email.domain.root_domain in $free_email_providers
+   and length(attachments) == 1
+   and all(attachments, .file_extension == "pdf"
+   and regex.icontains(.file_name, '(\d\.\d{3,4}.{0,12}BTC|bitcoin)')
+   and any(beta.binexplode(.), (.scan.exiftool.page_count >=5
+       and .depth == 0 and length(filter(.scan.strings.strings, regex.match(.,'^\/URI \(http.*'))) == 1))
```

```suggestion
       and regex.icontains(.file_name, '(\d\.\d{3,4}.{0,12}BTC|bitcoin)')
       and any(beta.binexplode(.),
           .scan.exiftool.page_count >=5
           and .depth == 0
           and length(filter(.scan.strings.strings, regex.match(.,'^\/URI \(http.*'))) == 1)
```

minor readability improvement